frontend/src/pages/WritingTablePage/WritingTablePage.tsx

```diff
@@ -1,18 +1,25 @@
 import { useQuery } from '@tanstack/react-query';
+import { useGlobalStateValue } from '@yogjin/react-global-state-hook';
 import { getDetailWritings } from 'apis/writings';
 import WritingTable from 'components/WritingTable/WritingTable';
+import { activeCategoryIdState } from 'globalState';
 import { useParams } from 'react-router-dom';
 import { styled } from 'styled-components';
 import { sidebarStyle } from 'styles/layoutStyle';
 
 const WritingTablePage = () => {
-  const categoryId = Number(useParams()['categoryId']);
-  const { data } = useQuery(['detailWritings', categoryId], () => getDetailWritings(categoryId));
+  const activeCategoryId = useGlobalStateValue(activeCategoryIdState);
+
+  if (!activeCategoryId) return <div>카테고리가 존재하지 않습니다.</div>;
+
+  const { data } = useQuery(['detailWritings', activeCategoryId], () =>
+    getDetailWritings(activeCategoryId),
+  );
 
   return (
     <S.Article>
       <S.CategoryNameTitle>{data?.categoryName}</S.CategoryNameTitle>
-      <WritingTable categoryId={categoryId} writings={data?.writings ?? []} />
+      <WritingTable categoryId={activeCategoryId} writings={data?.writings ?? []} />
```

Comment:
카테고리에 글 없을 경우 이렇게 나오는데
<img width="953" alt="image" src="https://github.com/woowacourse-teams/2023-dong-gle/assets/57815133/0a794a3a-8c66-429e-aa7a-1e374339f473">

뭔가 이렇게 안내 해주는 게 좋을 것 같아욥
```suggestion
      { 
      data?.writings ? 
        <WritingTable categoryId={activeCategoryId} writings={data?.writings} /> 
        : <p> 카테고리 내 글이 존재하지 않습니다 </p> 
      }
```